src/page/post_view/video_loader.rs

```diff
@@ -116,6 +120,69 @@ pub fn VideoView(idx: usize, muted: RwSignal<bool>) -> impl IntoView {
         Some(())
     });
 
+    let watched_percentage_signal = create_rw_signal(0_u8);
+    let watched_count_signal = create_rw_signal(0_u8);
+
+    let mut video_previous_current_time = 0.0;
```

Comment:
don't want to move outside the reactive system of leptos
https://docs.rs/leptos/latest/leptos/fn.store_value.html